Makefile

```diff
@@ -14,10 +14,9 @@ ifdef config
   endif
 endif
 
-ifeq ($(config),release)
-	PONYC = ponyc
-else
-	PONYC = ponyc --debug
+PONYC ?= ponyc
```

Comment:
i'm reverting this change.

Reply:
In general I am fine with this. I am just used to use stuff with a version of pony i do not have on the path. And overwriting the used `ponyc` from `PATH` by setting the environment variable `PONYC`, helped me a lot.